pkgs/by-name/la/lasso/package.nix

```diff
@@ -32,6 +32,15 @@ stdenv.mkDerivation rec {
       url = "https://git.entrouvert.org/entrouvert/lasso/commit/ffaddeb015a61db3e52c391de00430107a23e2f1.patch";
       hash = "sha256-D2npxpIuR/KrNYiKO3KXCvHEb/XVXUKIP0HQUd+w56k=";
     })
+    # Fix GCC 14 implicit declaration of function
+    ./fix-gcc14-implicit-function-declaration.diff
+    # Fix GCC 14 incompatible pointer
+    ./fix-gcc14-incompatible-pointer.diff
```

Comment:
Could you add a description of where these came from? Then we're good to go I think.